### DISCUSSION TITLE
[keyball61]使用できないLED発光パターンがあります

### DISCUSSION THREAD

**Discussion by kh-sakuya-1124** [Yowkees/keyball#452] [Q&A] [answered]
keyball61を使用しています。
デフォルトのレイヤー3に設定されているLED発光パターンの中に動作しないものがありますが、メモリ容量による制限でしょうか。
![image](https://github.com/Yowkees/keyball/assets/154493952/d9db14e8-13be-435d-9a5a-bdfb3f3f13a8)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

私もすべての発光パターンを認識できておらず恐縮ですが、"Mode Test"はviaファームウェアではメモリ容量を稼ぐために外してあると記憶しております。おそらくその他も同様の理由で使用できません。

以上、よろしくお願いいたします。
